docs/apis/ledgerstate.md

```diff
@@ -0,0 +1,1112 @@
+
+### Examples
+
+#### cURL
+
+```shell
+curl http://localhost:8080/ledgerstate/addresses/:address \
+-X GET \
+-H 'Content-Type: application/json'
+```
+#### client lib - `GetAddressOutputs()`
+```Go
+resp, err := goshimAPI.GetAddressOutputs("6PQqFcwarCVbEMxWFeAqj7YswK842dMtf84qGyKqVH7s1kK")
+	if err != nil {
+		// return error
+	}
+	fmt.Println("output address: ", resp.Address)
+	
+	for _, output := range resp.Outputs {
+		fmt.Println("outputID: ", output.OutputID)
+		fmt.Println("output type: ", output.Type)
+		// get output instance
+		out, err = output.ToLedgerstateOutput()
+```
+
+### Response examples
+```json
+{
+    "address": {
+        "type": "AddressTypeED25519",
+        "base58": "18LhfKUkWt4M9YR6Q3au4LT8wWCERwzHaqn153K78Eixp"
+    },
+    "outputs": [
+        {
+            "outputID": {
+                "base58": "gdFXAjwsm5kDeGdcZsJAShJLeunZmaKEMmfHSdoX34ZeSs",
+                "transactionID": "32yHjeZpghKNkybd2iHjXj7NsUdR63StbJcBioPGAut3",
+                "outputIndex": 0
+            },
+            "type": "SigLockedColoredOutputType",
+            "output": {
+                "balances": {
+                    "11111111111111111111111111111111": 1000000
+                },
+                "address": "18LhfKUkWt4M9YR6Q3au4LT8wWCERwzHaqn153K78Eixp"
+            }
+        }
+    ]
+}
+```
+
+### Results
+|Return field | Type | Description|
+|:-----|:------|:------|
+| `address`  | Address | The address corresponding to provided outputID.   |
+| `outputs`   | Output | List of transactions' outputs.     |
+
+`Address`
+
+|Field | Type | Description|
+|:-----|:------|:------|
+| `type`  | string | The type of an address.   |
+| `base58`| string   | The address encoded with base58.          |
+
+`Output`
+
+|Field | Type | Description|
+|:-----|:------|:------|
+| `outputID`  | OutputID | The identifier of an output.   |
+| `outputType`   | string | The full ID of a node.     |
+| `output` | string | An output raw message containing balances and corresponding addresses. |
+
+`OutputID`
+
+|Field | Type | Description|
+|:-----|:------|:------|
+| `base58`  | string | The output identifier encoded with base58.    |
+| `transactionID`   | string | The transaction identifier encoded with base58.     |
+| `outputIndex`   | int | The index of an output.     |
+
+## `/ledgerstate/addresses/:address/unspentOutputs`
+Gets list of all unspent outputs for the address based on a given base58 encoded address ID.
+
+### Parameters
+
+| **Parameter**            | `addressID`      |
+|--------------------------|----------------|
+| **Required or Optional** | required       |
+| **Description**          | The address ID encoded in base58. |
+| **Type**                 | string         |
+### Examples
+
+#### cURL
+
+```shell
+curl http://localhost:8080/ledgerstate/addresses/:address/unspentOutputs \
+-X GET \
+-H 'Content-Type: application/json'
+```
+#### client lib - `GetAddressUnspentOutputs()`
```

Comment:
This is hard to read in the rendered version. We should change this from 
#### client lib - `GetAddressUnspentOutputs()`
to a bullet point
* client lib - `GetAddressUnspentOutputs()`
for all examples (not only in this document)

Reply:
Don't we loose the links to the methods in the table of content?